modules/post/windows/gather/credentials/securecrt.rb

```diff
@@ -62,8 +62,13 @@ def try_encode_file(data)
   def enum_session_file(path)
     config_ini = []
     tbl = []
-    print_status("Searching for session files in #{path}")
-    config_ini += session.fs.file.search(path, '*.ini')
+    begin
+      print_status("Searching for session files in #{path}")
+      config_ini += session.fs.file.search(path, '*.ini')
+      fail_with(Failure::BadConfig, "Couldn't find any session files at #{path}") if config_ini.to_s.empty?
```

Comment:
`config_ini` is guaranteed to be an array right? I think you should just be able to do `config_ini.empty?` right? unless I'm missing something here
https://apidock.com/ruby/Array/empty%3F